ote_sdk/ote_sdk/usecases/exportable_code/prediction_to_annotation_converter.py

```diff
@@ -275,7 +275,7 @@ def convert_to_annotation(
         self, predictions: np.ndarray, metadata: Dict[str, Any]
     ) -> AnnotationSceneEntity:
         pred_score = predictions.reshape(-1).max()
-        pred_label = pred_score >= metadata.get("threshold", 0.5)
+        pred_label = pred_score >= metadata.get("threshold", 20)
```

Comment:
@akorobeinikov @ashwinvaidya17 @samet-akcay @djdameln 
this should be fixed as soon as confidence normalization issue is fixed

Reply:
fixed